src/main/java/HelpCommand.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Simulates a help command of the Duke System.
+ * @author Fabian Chia Hup Peng
+ */
+
+public class HelpCommand extends Command {
+
+    /**
+     * Executes the help command; shows all possible Duke commands.
```

Comment:
Having this Help command feature is great- It teaches the user how to use your program!